### DISCUSSION TITLE
[급] 빌드잇 로더 클릭 시 500 internal server error

### DISCUSSION THREAD

**Discussion by natural-jogi** [naverz/zepeto-studio-kor#153] [Issue Report (Legacy)]
![image](https://user-images.githubusercontent.com/67745779/147798942-49c3310a-9e6e-489c-9fd8-97ef79e8cd10.png)

빌드잇으로 작업한 맵 파일을 유니티로 import 시키려 빌드잇 로더를 클릭할 경우 위처럼 500 Internal Server Error가 뜨면서 맵이 불러와지지 않습니다.
작업이 진행되고 있지 않아 난감한 상황입니다... 확인 부탁드립니다.

**Comment by x4cf** [comment]
일시적인 서버오류로 지금은 모두 해결된것으로 확인됩니다 :)

**Comment by hyeonjin-jo** [comment]
안녕하세요.

일시적인 서버 오류로 인해서 발생한 문제입니다.

이후 반복되지 않도록 체크 진행하겠습니다

감사합니다.
